File: RLE.API/Models/Department.cs
using System;
using System.Collections.Generic;

namespace RLE.API.Models
{
    public partial class Department
    {
        
        public int Id { get; set; }
        public string Name { get; set; }
        public int? RegionId { get; set; }
        public string Code { get; set; }
        public byte Active { get; set; }
        public bool ActiveforInscription { get; set; }

        public  Region Region { get; set; }
        public virtual  ICollection<City> Cities { get; set; }
        // public  ICollection<User> Employee { get; set; }
        // public  ICollection<TrainingClass> TrainingClass { get; set; }
    }
}

File: RLE-SPA/src/app/_models/department.ts
import { Region } from './region';
import { City } from './city';

export class Department {
  id: number;

  name: string;

  regionId: number;

  code: string;

  active: number;
  totalOnTraining: number;
  totalPreSelected: number;
  totalRegistered: number;
  totalSelected: number;
  nbEmpNeeded: number;
  prctRegistered: number;
  prctPreselected: number;
  prctOnTraining: number;
  prctSelected: number;
  activeforInscription: boolean;
  totalRegisteredPrct: number;
  totalReserveToHave:number;

  Region: Region;
  cities: City[];

}
